src/XMakeTasks/UnitTests/GenerateResource_Tests.cs

```diff
@@ -2071,6 +2071,7 @@ public void SourceItemMissing()
 #else
         [Fact(Skip = "https://github.com/Microsoft/msbuild/issues/297")]
 #endif
+        [PlatformSpecific(Xunit.PlatformID.Windows)]
```

Comment:
This test creates a read-only (removes the write permission) state file and then expects the task to fail while trying to write to such a file. But the [code](https://github.com/Microsoft/msbuild/blob/xplat/src/XMakeTasks/StateFileBase.cs#L40) (`GenerateResource.WriteStateFile -> ResGenDependencies.SerializeCache -> StateFileBase.SerializeCache`) that does this, deletes the file before writing to it.
- On Unix, it is legal to delete such a file. So, subsequently trying to write to that path succeeds and the test fails
- IIUC, trying to delete or write to this read-only would fail on windows, thus satisfying the test.

I should add this to the commit message.